docs/UserGuide.md

```diff
@@ -38,97 +38,204 @@
-> * Only the name is searched.
-> * Only full words will be matched e.g. `Han` will not match `Hans`
-> * Persons matching at least one keyword will be returned (i.e. `OR` search).
-    e.g. `Hans` will match `Hans Bo`
+> Selects the task and loads the details of the task at the specified `INDEX`. 
+  The index refers to the index number shown in the most recent listing.<br>
+  The index **must be a positive integer** 1, 2, 3, ...
+> Allow multiple selection
 
 Examples: 
-* `find John`<br>
-  Returns `John Doe` but not `john`
-* `find Betsy Tim John`<br>
-  Returns Any person having names `Betsy`, `Tim`, or `John`
+* `list`<br>
+  `select 2`<br>
+  Selects the 2nd task in the task book.
+* `find CS2103` <br> 
+  `select 1`<br>
+  Selects the 1st task in the results of the `find` command.
 
-#### Deleting a person : `delete`
-Deletes the specified person from the address book. Irreversible.<br>
-Format: `delete INDEX`
+#### Modifies a task : `modify`
+Marks the task identified by the index number used in the last task listing.<br>
+Format: `modify INDEX [t/TASK_NAME] [s/START_DATE] [st/START_TIME] [e/END_DATE] [et/END_TIME] [l/LOCATION] [p/PRIORITY_LEVEL] [r/RECURRING_TYPE] [n/NUMBER_OF_RECURRENCE] [c/CATEGORY] [d/DESCRIPTION]`
 
-> Deletes the person at the specified `INDEX`. 
+> Selects the task and modifies the task as done at the specified `INDEX`. 
+  The index refers to the index number shown in the most recent listing.<br>
+  The index **must be a positive integer** 1, 2, 3, ...
+> Overwrites any of the specified fields ('LOCATION', 'DESCRIPTION'...) with the new values
+
+#### Mark a task as done : `mark`
+Marks the task as completed identified by the index number used in the last task listing. Completed task will be remove from the normal list and placed under archived list<br>
+Format: `mark INDEX [MORE_INDEX]`
+
+> Selects the task and marks the task as done at the specified `INDEX`. 
   The index refers to the index number shown in the most recent listing.<br>
   The index **must be a positive integer** 1, 2, 3, ...
 
 Examples: 
 * `list`<br>
-  `delete 2`<br>
-  Deletes the 2nd person in the address book.
-* `find Betsy`<br> 
-  `delete 1`<br>
-  Deletes the 1st person in the results of the `find` command.
+  `mark 2`<br>
+  Marks the 2nd task in the task book as completed.
+* `find CS2103` <br> 
+  `mark 1`<br>
+  Marks the 1st task in the results of the `find` command as completed.
 
-#### Select a person : `select`
-Selects the person identified by the index number used in the last person listing.<br>
-Format: `select INDEX`
+#### Unmark a task as done : `unmark`
+Unmarks the task identified by the index number used in the last task listing.<br>
+Format: `Unmark INDEX [MORE_INDEX]`
 
-> Selects the person and loads the Google search page the person at the specified `INDEX`. 
+> Selects the task and marks the task as done at the specified `INDEX`. 
```

Comment:
may be you need to be specific here, to tell list the archived tasks and then unmark? since archived tasks are not shown in the default list, unmark index in the current view will mane little sense, as that task is not completed anyway. no?